src/win/fs.c

```diff
@@ -245,6 +245,7 @@ INLINE static void uv_fs_req_init(uv_loop_t* loop, uv_fs_t* req,
   req->ptr = NULL;
   req->path = NULL;
   req->cb = cb;
+  req->fs.info.bufs = NULL;
   memset(&req->fs, 0, sizeof(req->fs));
```

Comment:
You set part of the `req->fs` struct to `NULL` and then immediately `memset` it whole to zero?  Am I missing something?

Reply:
You are, in fact, not missing something. My mistake. Thanks for catching it. 😄